test/spec/LoginRouter_spec.js

```diff
@@ -533,6 +536,38 @@ function (Okta, Q, Backbone, xdomain, SharedUtil, OktaAuth, Util, Expect, Router
       });
     });
 
+    describe('Events', function () {
+      itp('triggers a pageRendered event when first controller is loaded', function() {
+        return setup()
+        .then(function (test) {
+          test.router.primaryAuth();
+          return tick(test)
+              .then(function(test){
+                expect(test.eventSpy.calls.count()).toBe(1);
+                expect(test.eventSpy).toHaveBeenCalledWith('pageRendered', { page: 'primary-auth'});
+              });
+        });
+      });
+      itp('triggers a pageRendered event when navigating to a new controller', function() {
+        return setup()
+            .then(function (test) {
+              // Test navigation from primary Auth to Forgot password page
+              test.router.primaryAuth();
+              Util.mockRouterNavigate(test.router);
+              test.router.navigate('signin/forgot-password');
```

Comment:
Since we're leaving a page, it's nice to do something like (before test.router.navigate):
```javascript
// reset call count
test.eventSpy.calls.reset();
```
This might help you hone in on why it's triggering more events than you expect (and makes for a cleaner test, where the next assertion should be that the call count is "1" again.